docs/inference-providers/guides/github-actions-code-review.md

```diff
@@ -0,0 +1,182 @@
+# Automating Code Review with GitHub Actions
+
+[OpenCode](https://opencode.ai) is an AI coding agent that runs in your terminal and can work with open models via Hugging Face Inference Providers.
+
+You can also install OpenCode as a GitHub App to help automate GitHub workflows!
+
+In less than 5 minutes, you can set it up to respond to issues and pull requests using powerful open source language models.
+
+This guide shows you how to use Hugging Face Inference Providers with OpenCode to power GitHub automation. You'll be able to triage issues, implement features, and review code using models like DeepSeek, GLM-4.5, and Kimi K2.
+
+> [!TIP]
+> This guide assumes you have a Hugging Face account and a GitHub repository. You can create a free Hugging Face account at [huggingface.co](https://huggingface.co).
+
+## What the OpenCode GitHub App Does
+
+Once set up, OpenCode will respond to `/oc` or `/opencode` commands in GitHub issues and pull requests:
+
+- **Triage and explain issues** - Ask OpenCode to analyze an issue and explain it
+- **Implement features** - Request a fix or feature, and OpenCode creates a PR with the changes
+- **Review and modify PRs** - Request changes on pull requests and OpenCode commits them
+
+The entire workflow runs securely in your GitHub Actions runners, and you maintain full control over which repositories you install the OpenCode GitHub app on.
+
+## Step 1: Install OpenCode
+
+First, install OpenCode on your local machine see the [installation instructions](https://opencode.ai/docs/#install).
+
+## Step 2: Configure OpenCode for GitHub
+
+Navigate to your local clone of your GitHub repository and run the setup command:
+
+```bash
+cd your-repository
+opencode github install
+```
+
+This interactive setup will guide you through:
+
+1. **Installing the GitHub app** - Your browser opens to authorize OpenCode for your repositories. You can choose specific repositories or all repositories.
+2. **Selecting a provider** - Choose **Hugging Face** from the list
+3. **Choosing a model** - Select a model like **GLM-4.5-Air** or **Kimi-K2-Instruct**
+4. **Creating the workflow** - Generates `.github/workflows/opencode.yml` automatically
+
```

Comment:
Maybe here a screenshot of what to except from local opencode in terminal?

Reply:
Yes, agree a screenshot could be illustrative.